develop/interoperability/test-and-debug.md

```diff
@@ -9,22 +9,36 @@ description: Learn how to test and debug cross-chain communication via the XCM S
 
 Cross-Consensus Messaging (XCM) is a core feature of the Polkadot ecosystem, enabling communication between parachains, relay chains, and system chains. To ensure the reliability of XCM-powered blockchains, thorough testing and debugging are essential before production deployment.
 
-This article explores two indispensable tools for XCM testing, the [XCM Simulator](#xcm-simulator) and  the [XCM Emulator](#xcm-emulator), to help developers onboard and test their solutions effectively.
+This guide focuses on the XCM Emulator, a tool designed to help developers onboard and test their solutions effectively. For additional XCM testing capabilities, you can also use [Chopsticks](/tutorials/polkadot-sdk/testing/fork-live-chains/#xcm-testing){target=\_blank}.
```

Comment:
I would point them first to chopsticks for testing and debugging and tell them how to do both.
I would mention how to use `setupNetworks` from `@acala-network/chopsticks-testing` to write automatic tests like in https://github.com/franciscoaguirre/dot-reserve-change/blob/2b678660aa9c5c10c51ec7c58cd872dd224d067b/setup.ts#L31.

Reply:
This is really useful and the way to go. For the context of this PR, it is better to update the XCM emulator docs and create an issue to add chopsticks xcm testing docs later. We already have some documentation on Chopsticks, so we need to reorganize the informational architecture to include these changes seamlessly.

Once we have the IA done, we can iterate over this again